packages/cloud_firestore/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.7.0
+
+* **Breaking change**. snapshots is now a method instead of a getter.
+* **Breaking change**. setData uses optional arguments instead of SetOptions.
```

Comment:
Maybe use code font on `snapshots`, `setData`, and `SetOptions` to avoid having the entry look like a spelling error with lowercase word following the full stop.

[nit] optional arguments => named argument?

Reply:
Done